src/SWP/Bundle/AnalyticsBundle/Repository/ArticleEventRepository.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Superdesk Web Publisher Analytics Bundle.
+ *
+ * Copyright 2018 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2018 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\AnalyticsBundle\Repository;
+
+use SWP\Bundle\AnalyticsBundle\Model\ArticleEventInterface;
+use SWP\Bundle\ContentBundle\Model\ArticleInterface;
+use SWP\Bundle\StorageBundle\Doctrine\ORM\EntityRepository;
+
+class ArticleEventRepository extends EntityRepository implements ArticleEventRepositoryInterface
+{
+    public function getCountForArticleInternalPageViews(ArticleInterface $article): int
```

Comment:
What's the difference between internal page view and page views? In entity class I see it's `page views` but here it's internal page views. It's a bit confusing.

Reply:
Article event have aslo page view source. Internal is one coming from our website, external is google, other website links, or direct traffic.